frontend/src/App/App.tsx

```diff
@@ -58,22 +60,34 @@ function App() {
                          orders={orders}
                          updateBikeList={updateBikeList}
                          updateOrderList={updateOrderList}
-                         workshops={workshops}/>}/>
+                         workshops={workshops}
+                         mapApiKey={mapApiKey}/>}/>
           <Route path={"/bikes"} element={<BikeGallery bikes={bikes}/>}/>
           <Route path={"/bikes/add-bike"} element={<AddBikePage bikes={bikes} updateBikeList={updateBikeList}/>}/>
           <Route path={"/bikes/details/:bikeId"} element={<BikeDetailPage bikes={bikes}/>}/>
           <Route path={"/bikes/edit-bike/:bikeId"} element={
               <EditBikePage bikes={bikes} updateBikeList={updateBikeList}/>}/>
           <Route path={"/workshops/orders/:workshopId"} element={
-              <BookOrderPage workshops={workshops} bikes={bikes} orders={orders} updateOrderList={updateOrderList}/>}/>
+              <BookOrderPage workshops={workshops}
+                             bikes={bikes}
+                             orders={orders}
+                             updateOrderList={updateOrderList}
+                             mapApiKey={mapApiKey}/>}/>
```

Comment:
Maybe you want to consider using a context for data that is required in a lot of components. But as so often I guess it's more a question of personal preference :)